### PR TITLE
feat(ChangeLog): Highlight changes between old and new revision of Document

### PR DESCRIPTION
**Pull request by JaideepPalit**
> Highlight changes between old and new revision of Document
> * Which issue is this pull request belonging to and how is it solving it? (*#1118*)
> * Did you add or update any new dependencies that are required for your change? - No

> Have you implemented any additional tests? - No

Object
StringCapture
Array

### Checklist
Must:
- [X] All related issues are referenced in commit messages and in PR

Signed-off-by: Jaideep Palit <jaideep.palit@siemens.com>

### CODE REVIEW

File: frontend/sw360-portlet/src/main/resources/META-INF/resources/html/changelogs/elementView.jsp [hunk 1]
```diff
@@ -218,25 +217,227 @@
...
+                    j = 0,
+                    result = "";
+
+                while(i < fieldValuePrimaryArr.length) {
+                    if(fieldValuePrimaryArr[i] === fieldValueSecondaryArr[j]) {
+                        result = prepareDiffString(changesArrTmp, result, highLighter, fieldValuePrimaryArr[i]);
+                        j++;
+                    } else {
+                        let index = fieldValueSecondaryArr.indexOf(fieldValuePrimaryArr[i]);
+                        if (index ===  -1){
```
Comment (smrutis1) [inline frontend/sw360-portlet/src/main/resources/META-INF/resources/html/changelogs/elementView.jsp:417]: Indentation

**Review by smrutis1 — COMMENTED** [review]
Tested the highlighting feature for some of the fields, working fine .

**Comment by JaideepPalit** [comment]
Thanks for review and test. Fixed the indentation.

**Review by smrutis1 — APPROVED** [review]
Code looks good to me. Tested again successfully.

**Pull request merged**